smartsim/_core/_cli/validate.py

```diff
@@ -241,9 +241,9 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
 
     net = Net()
     forward_input = torch.rand(1, 1, 3, 3)
-    traced = torch.jit.trace(net, forward_input)  # type: ignore[no-untyped-call]
+    traced = torch.jit.trace(net, forward_input)
```

Comment:
Removing the comment on this line and on line 246 makes `mypy` fail. Was there a reason for doing this?